src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -324,8 +346,8 @@ function ComposerWithSuggestions({
         [suggestionsRef],
     );
 
-    const updateShouldShowSuggestionMenuToFalse = useCallback(() => {
-        if (!suggestionsRef.current) {
+    const hideSuggestionMenu = useCallback(() => {
```

Comment:
I really like this name change, it's like changing `giveCatLikeHighPitchedSound` to `meow`